rfi_file_monitor/utils.py

```diff
@@ -103,6 +112,8 @@ def update_from_dict(self, yaml_dict: dict):
                         widget.set_text("")
                     else:
                         widget.set_text(value)
+                elif isinstance(widget, Gtk.ComboBoxText):
+                    widget.set_active("")
```

Comment:
This is incorrect, as `set_active` takes an int to specify the n-th item in the underlying `ListStore` you want to activate. 

For this, you will need to iterate over all items in the `ListStore` and check which one matches the value.

There is unfortunately no `set_active_text` method 😢

Reply:
I have changed this Combobox to append an updated value